File: python/bee1165.py
n = int(input())

for i in range(n):
    divisores = 0
    x = int(input())

    for j in range(1, x+1):
        if x % j == 0:
            divisores += 1
    
    if divisores > 2:
        print(f'{x} nao eh primo')
    else:
        print(f'{x} eh primo')

File: java/bee1165.java
import java.util.Scanner;

public class bee1165 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int N = input.nextInt();

        for(int i = 0; i < N; i++) {
            int div = 0;
            int number = input.nextInt();

            for(int j = 1; j <= number; j++) {
                if(number % j == 0) {
                    div++;
                }
            }

            if(div > 2) {
                System.out.println(number + " nao eh primo");
            } else {
                System.out.println(number + " eh primo");
            }
        }

        input.close();
    }
}
